www/zemk05/zivotopis/css/styles.css

```diff
@@ -0,0 +1,88 @@
+*{
+    box-sizing: border-box;
+    background-repeat: no-repeat;
+}
+
+body {
+    background-image: url('https://imgnooz.com/sites/default/files/wallpaper/abstract/55356/brown-background-wallpapers-55356-4029363.jpg');
```

Comment:
link na externi obrazek nemusi byt vzdy dostupny a nemas nad nim kontrolu, proto je lepsi si ho stahnout k sobe,

Reply:
Můžu se zeptat zda to mám celé změnit a udělat img src v html? Protože když upravím tento background-image na cestu k souboru, celé stylování se mi rozbije, ačkoliv je tam vše nastavené, obrázky se tam naházejí jak chtějí

Reply:
nene, nedávej to do img src... jakože ty obrázky nejsou vidět, protože se nenašly? jakým způsobem jsi upravila url()?

Reply:
obrázky jsem dala do speciální složky v css file, všechno kromě background image na body se nakonec spravilo. Nevím proč, ale u jediného body to má problém se všemi třemi příkazy, prý mu tam někde něco chybí, zbytek dokumentu je v pořádku. Ten background u tohohle elementu není vidět:
body{
    background-image: url("images/brown-background.jpg")
    background-size: cover;
    font-family:Georgia,'Times New Roman', Times, serif;
}

Reply:
středník na konci prvního stylu?

jinak složku images dej o jednu úroveň výš tak jak je složka css (nedávej ji do vnitř css) a pak v url() napíšeš '../images/brown-backgorund.jpg'

Reply:
Děkuji, nevěděla jsem jak vyjádřit tu nadřazenou složku. Posílám upravenou verzi.